tools/release/01_bump_crate_versions.ts

```diff
@@ -101,3 +112,26 @@ async function getGitLog() {
     );
   }
 }
+
+async function updateStdVersion() {
+  const newStdVersion = await getLatestStdVersion();
+  const compatFilePath = path.join(cliCrate.folderPath, "compat/mod.rs");
+  const text = Deno.readTextFileSync(compatFilePath);
+  Deno.writeTextFileSync(
+    compatFilePath,
+    text.replace(/std@[0-9]+\.[0-9]+\.[0-9]+/, `std@${newStdVersion}`),
+  );
+}
+
+async function getLatestStdVersion() {
+  const url =
+    "https://raw.githubusercontent.com/denoland/deno_std/main/version.ts";
```

Comment:
Not for now, but we should maybe rename this to `version.json`, then you wouldn't need the regex.